tools/assetprocessor/main.cpp

```diff
@@ -57,9 +57,11 @@ int main(int argc, char *argv[])
     QCommandLineOption inputOption({ "i", "input" }, QObject::tr("glTF2 <file> to compress."), QObject::tr("file"), QString());
     QCommandLineOption outputOption({ "o", "output" }, QObject::tr("Output <directory>."), QObject::tr("directory"), QString());
     QCommandLineOption ratioOption({ "q", "quantization" }, QObject::tr("Compression quantization, between 1 (more compressed) and 16 (less compressed)."), QObject::tr("value"), "7");
+    QCommandLineOption embedOption({ "e", "embed" }, QObject::tr("Embed assets in base64."));
+    QCommandLineOption extractOption({ "x", "extract" }, QObject::tr("Extract embedded assets."));
```

Comment:
I would suggest `separate` rather than `extract`. Also need to indicate what the default is.

Options should be documented in the readme file and maybe as qdoc?

Reply:
should I make a readme specific for this one or just add it to the main readme ?